### PR TITLE
Added hotgraphic tile hover animation back in

### PR DESCRIPTION
**Pull request by joe-replin**
I've used this layout lately and noticed that earlier framework versions had a hover animation. This is my proposal to add it back in.

### CODE REVIEW

File: less/plugins/adapt-contrib-hotgraphic/hotgraphic.less [hunk 1]
```diff
@@ -26,4 +26,39 @@
...
   }
   // --------------------------------------------------
+  // _useGraphicsAsPins config
+  // --------------------------------------------------
+  // _use
+  &__tile-item {
+    position: relative;
+    z-index: 4;
+
+    .hotgraphic__tile {
```
Comment (guywillis) [inline less/plugins/adapt-contrib-hotgraphic/hotgraphic.less:36]: Suggest moving the nested styles outside of the tile item element. can also make use of `&__` so don't have to repeat the word `hotgraphic`

```
&__tile:before {
  content: '';
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background-color: fadeout(@item-color, 100%);
  .transition(background-color @duration ease-in);
}
```

File: less/plugins/adapt-contrib-hotgraphic/hotgraphic.less [hunk 2]
```diff
@@ -26,4 +26,39 @@
...
+        top: 0px;
+        left: 0px;
+        width: 100%;
+        height: 100%;
+        background: fadeout(@item-color, 100%);
+        transition: background @duration ease-in;
+      } 
+    }
+
+    .no-touch &:hover {
```
Comment (guywillis) [inline less/plugins/adapt-contrib-hotgraphic/hotgraphic.less:49]: Suggest removing the nesting

```
.no-touch &__tile-item:hover {
  z-index: 5;
}
```

File: less/plugins/adapt-contrib-hotgraphic/hotgraphic.less [hunk 3]
```diff
@@ -26,4 +26,39 @@
...
+      z-index: 5;
+      
+      .hotgraphic__tile {
```
Comment (guywillis) [inline less/plugins/adapt-contrib-hotgraphic/hotgraphic.less:52]: Suggest removing the nesting, utilising the inbuilt transform mixin, and using multiple ampersand symbols.

```
.no-touch &__tile-item:hover &__tile {
  box-shadow: 0 2px 5px fade(@black, 20%);
  .transform(scale(1.03));
  .transition(transform @duration ease-in, box-shadow @duration ease-in;);
}
```

File: less/plugins/adapt-contrib-hotgraphic/hotgraphic.less [hunk 4]
```diff
@@ -26,4 +26,39 @@
...
+        -webkit-transform: scale(1.03);
+        transform: scale(1.03);
+        box-shadow: 0 2px 5px fade(@black, 20%);
+        .transition(transform @duration ease-in, box-shadow @duration ease-in;);
+
+        &:before {
```
Comment (guywillis) [inline less/plugins/adapt-contrib-hotgraphic/hotgraphic.less:58]: Suggest removing the nesting and utilising the ampersand

```
.no-touch &__tile-item:hover &__tile:before {
  background: fadeout(@item-color, 85%);
}
```

**Comment by oliverfoster** [comment]
stale

**Pull request closed without merging**